gubernator/main.py

```diff
@@ -34,11 +35,8 @@
 import filters
 
 BUCKET_WHITELIST = {
-    'kubernetes-jenkins',
-    'kube_azure_log',
-    'rktnetes-jenkins',
-    'kubernetes-github-redhat',
-    'kopeio-kubernetes-e2e',
+    re.match(r'gs://([^/]+)', path).group(1)
+    for path in yaml.load(open("buckets.yaml"))
```

Comment:
This should read a live file. We shouldn't have to push gubernator to update the list of buckets it scans